utilities/cli/util.py

```diff
@@ -1128,6 +1154,7 @@ def collect_environment_variables() -> None:
         "HOLOHUB_DEFAULT_DOCKERFILE",
         "HOLOHUB_BASE_IMAGE_FORMAT",
         "HOLOHUB_DEFAULT_IMAGE_FORMAT",
+        "HOLOHUB_DEFAULT_BUILD_ARGS",
```

Comment:
Should that be documented? (I guess like all other vars, having a table with existing env vars would be great)

Reply:
sure, will have a follow up PR on the docs